RELEASE-NOTES.md

```diff
@@ -13,6 +13,8 @@
 - The `pymc3.memoize` module was removed and replaced with `cachetools`.  The `hashable` function and `WithMemoization` class were moved to `pymc3.util` (see [#4509](https://github.com/pymc-devs/pymc3/pull/4509)).
 - Remove float128 dtype support (see [#4514](https://github.com/pymc-devs/pymc3/pull/4514)).
 - `pm.make_shared_replacements` now retains broadcasting information which fixes issues with Metropolis samplers (see [#4492](https://github.com/pymc-devs/pymc3/pull/4492)).
+- Logp method of `Uniform` and `DiscreteUniform` no longer depends on `pymc3.distributions.dist_math.bound` for proper evaluation (see [#4541](https://github.com/pymc-devs/pymc3/pull/4541)).
```

Comment:
Something is wrong here as we already release `3.10.2` this should go into `vNext`.

Reply:
Right, it seems the release notes were only changed in V3. Should I rebase the release notes from here? https://github.com/pymc-devs/pymc3/blob/v3/RELEASE-NOTES.md

Reply:
Good question. The Release notes are the only document that spans branches and will continue to even after `v4` was released.
I don't think we'll be able to keep up with rebasing it going into the future.

Can you bundle these items in a `vNext (4.0.0)` section, but still point out those that were already released in `3.11.2`? You can link to the Release itself, where the release notes are duplicated.

Reply:
Sure, let me give it a try